### ISSUE TITLE
Meteor 1.0.4.x error: "While loading plugin `compileSass` from package `stevezhu:sass`: ... Cannot find module"

### ISSUE THREAD

**Issue by seeekr** [stevezhu/meteor-sass#4]
Steps to reproduce:
`meteor create something`
`cd something && meteor add stevezhu:sass && meteor update --release 1.0.4.1 && meteor`

And this is the error that I am getting then:

```
=> Errors prevented startup:                  
   
   While loading plugin `compileSass` from package `stevezhu:sass`:
   module.js:338:15: Cannot find module
   '/Users/denis/.meteor/packages/stevezhu_sass/.1.3.7.s5ffcc++os.linux.x86_64+os.osx.x86_64+web.browser+web.cordova/plugin.compileSass.os.osx.x86_64/npm/compileSass/node_modules/files.js'
   at Function.Module._resolveFilename (module.js:338:15)
   at Function.Module._load (module.js:280:25)
   at Module.require (module.js:364:17)
   at require (module.js:380:17)
   at Object._.extend.Npm.require (/Users/denis/.meteor/packages/meteor-tool/.1.0.43.585c0y++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/tools/bundler.js:1341:22)
   at packages/compileSass/plugin/compile-sass-utils.js:5:1
   at <runJavaScript-13>:148:4
   at <runJavaScript-13>:329:3
   
   
=> Your application has errors. Waiting for file change.
```
(On OS X 10.10; tried with iojs 1.5.x as well as node 0.12.x and 0.10.36.)

A workaround for now is to just keep using Meteor 1.0.3.x, the command line for downgrading from 1.0.4.x (or any version) is `meteor update --release 1.0.3.2`.

Any ideas if this is originating from your/this package or from somewhere else and/or what the fix might be? As far as I can tell the problem might be related to a change in how `Npm.require` works now / where it looks for files or so... it seems that the Meteor guys are making lots of changes to improve Windows support and so they may have changed or fixed or cleaned up something about how some of the includes/include paths work... But that's just more or less a guess, maybe it helps point in a useful direction.

Thanks & appreciate making the sass package in the first place!

**Comment by flyrev** [comment]
I can confirm that I have the same issue and that the same solution worked. A fix would be much appreciated.

**Comment by stevezhu** [comment]
Working on a fix. It does seem to be because of changes with how Npm.require works with meteor tools.

**Comment by seeekr** [comment]
Great, thanks for the effort!

**Comment by stevezhu** [comment]
Sorry for the wait. I had to rewrite the package to remove some weird ideas I had that didn't really work with meteor. Everything should be working fine now on version 2.0.0.

**Comment by seeekr** [comment]
No worries about taking your time. Awesome it's fixed, upgrading my stuff to Meteor 1.4.x now :-)
